examples/chess/README.md

```diff
@@ -1,23 +1,76 @@
 # ZK Checkmate
 
-Proves that a chess position has a checkmate without revealing what that checkmate is.
+This code demonstrates a minimal example of how to use the RISC Zero [zkVM] to make ZK proofs about chess.
 
-Takes a move that is kept private and an initial board state that is published to the [journal](https://www.risczero.com/docs/explainers/zkvm/). There are default values for these, but you can also pass in an alternative move and/or board state as command line arguments. The move is in SAN format and the board state is in FEN format.
+The demo uses the [shakmaty] crate to prove that a chess position has a checkmate without revealing what that checkmate is.
 
-## Quick Start
+[zkVM]: https://dev.risczero.com/zkvm
+[shakmaty]: https://docs.rs/shakmaty/latest/shakmaty/
 
-First, make sure [rustup](https://rustup.rs) is installed. This project uses a [nightly](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) version of [Rust](https://doc.rust-lang.org/book/ch01-01-installation.html). The [`rust-toolchain`](rust-toolchain) file will be used by `cargo` to automatically install the correct version.
 
-To build all methods and create a zero-knowledge proof that the position defined [here](https://github.com/risc0/risc0/blob/main/examples/chess/src/main.rs#L28) includes a mate-in-one, run the following command from this directory:
+## Quick Start
 
+First, [install Rust] if you don't already have it.
```

Comment:
Oh; just realized that this drops the notes about `nightly` and `rust-toolchain`, which we probably want to keep (and should add to `bevy`).

Reply:
And maybe we want to be encouraging `rustup` still?

Reply:
We will need to have:
```
cargo install cargo-risczero
cargo risczero install
```